### PR TITLE
Edited main.py and test.py

### PR DESCRIPTION
**Pull request by amit-62**

### CODE REVIEW

File: fri/server/main.py [hunk 1]
```diff
@@ -174,7 +188,9 @@ def getFilesList(dir):
 
 @app.route('/openJupyter/', methods=['POST'])
 def openJupyter():
-    cur_path = os.getcwd()
+    # cur_path = os.getcwd()
+    cur_path = os.path.dirname(os.path.abspath(__file__))
+    print(cur_path)
```
Comment (pradeeban) [inline fri/server/main.py:193]: I know you added this line
    print(cur_path)
for demo purposes. Please remove this line after today's demo.

File: fri/test.py [hunk 2]
```diff
@@ -73,13 +67,24 @@ def openJupyter():
...
-# debug()
-# destroy()
-getFilesList()
+
+upload(files)
+execute()
+build("test", "sample1", "xyz")
+debug("sample1")
+destroy("sample1")
+# getFilesList("fri")
```
Comment (pradeeban) [inline fri/test.py:87]: Why are these two lines commented out?

**Review by pradeeban — CHANGES_REQUESTED** [review]
Pls address the comments.

File: fri/test.py [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
 
-  path = os.path.abspath("example.py")
+def upload(files):
+  url = "http://127.0.0.1:5000/upload/test?apikey=xyz"
 
   payload={}
-  files=[
-    ('files[]',('example.py',open(path,'rb'),'application/octet-stream'))
-  ]
+  # files=[
```
Comment (pradeeban) [inline fri/test.py:13]: Why are these commented out?

File: fri/test.py [hunk 4]
```diff
@@ -73,13 +70,30 @@ def openJupyter():
...
-# execute()
-# download()
-# build()
-# debug()
-# destroy()
-getFilesList()
+# file list to be uploaded
+files=[
+  #('files[]',(file_name,open(file_path,'rb'),'application/octet-stream'))
+  ('files[]',('controller.py',open('/home/amit/Desktop/test_xyz/controller.py','rb'),'application/octet-stream')),
```
Comment (pradeeban) [inline fri/test.py:81]: Pls do not hard-code the values such as "/home/amit" as these will not work outside your computer. Use relative paths instead.

Comment (pradeeban) [reply fri/test.py:81]: Replace '/home/amit/Desktop/test_xyz/controller.py' with '../demo/controller.py'

File: fri/server/main.py [hunk 5]
```diff
@@ -114,33 +79,75 @@ def build(dir):
...
+    else:
+        resp = jsonify({'message': 'There is an Error'})
+        resp.status_code = 500
+        return resp
+
+# to download /download/<dir>?fetch=<downloadfile>. For example, /download/test?fetchDir=xyz&fetch=u
 @app.route('/download/<dir>', methods=['POST', 'GET'])
 def download(dir):
     download_file = request.args.get('fetch')
-    apikey = request.args.get('apikey')
```
Comment (pradeeban) [inline fri/server/main.py:134]: Why are you removing apikey here?

**Review by pradeeban — CHANGES_REQUESTED** [review]
Please do not remove the "apikey" in the request path of methods.

File: fri/server/main.py (hunk 5 above)
Comment (amit-62) [reply fri/server/main.py:134]: we are downloading file from sample1 folder. which is created when we call build and debug.

**Review by pradeeban — APPROVED** [review]

**Pull request merged**